### DISCUSSION TITLE
0.56.2

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#8159] [General]
* Include packaged dxcore library to fix issues on retail Win11 [GH 8145]
* Mount drvfs shares on demand to avoid using the linked token. This improves startup time of launching WSL elevated and ensures the correct package context is used.
* Fix issue preventing launching Windows executables with large environments [GH 7947]
* Update kernel to 5.10.102.1  
    * Fix a new Linux kernel vulnerability called Dirty Pipe (CVE-2022-0847)
    * Disable unprivileged BPF by default
        * It can be re-enabled by setting `/proc/sys/kernel/unprivileged_bpf_disabled` to 0
    * Update Dxgkrnl Version to 2216
        * Fix out of bounds array access for ioctls[]
        * Implement wait for sync VM bus messages as killable to allow killing a process waiting for a synchronous call to the host
        * Flush device for termination when process is destroyed to avoid a deadlock when the guest process is killed

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/0.56.2'>0.56.2</a>.</em>

**Comment by benyaminl** [comment]
No update anymore for Win10 then?

**Reply by Biswa96** [reply]
It is possible to use the kernel file in Windows 10.
